src/EnergyPlus/SystemAvailabilityManager.hh

```diff
@@ -64,131 +65,122 @@ struct EnergyPlusData;
 
 namespace SystemAvailabilityManager {
 
-    int constexpr StayOff = 0;
-    int constexpr CycleOnAny = 1;
-    int constexpr CycleOnControlZone = 2;
-    int constexpr ZoneFansOnly = 3;
-    int constexpr CycleOnAnyCoolingOrHeatingZone = 4;
-    int constexpr CycleOnAnyCoolingZone = 5;
-    int constexpr CycleOnAnyHeatingZone = 6;
-    int constexpr CycleOnAnyHeatingZoneFansOnly = 7;
+    enum class ControlAlgorithm
+    {
+        Invalid = -1,
+        ConstantTemperatureGradient,
+        AdaptiveTemperatureGradient,
+        AdaptiveASHRAE,
+        ConstantStartTime,
+        Num
+    };
```

Comment:
Good stuff.

Reply:
And it looks like there are some that were removed. Is that correct?

Reply:
The `int constexpr` "groups" got rearranged, that's why it looks weird here...

Reply:
This enum class was previously the `int constexprs` [here.](https://github.com/NREL/EnergyPlus/blob/0210c3f801fa49d5936cc1fca367d19ae971da85/src/EnergyPlus/SystemAvailabilityManager.cc#L118-L121)